go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -1179,49 +1179,13 @@ func main() {
 	udp := flag.Bool("udp", false, "use udp transport protocol")
 	addressFamily := flag.String("addr-family", ip4, "Address family (ip4 or ip6) to be used for tracing")
 	skipOvnDetrace := flag.Bool("skip-detrace", false, "skip ovn-detrace command")
+	dumpVRFTableIDs := flag.Bool("dump-udn-vrf-table-ids", false, "Dump the VRF table ID per node for all the user defined networks")
 	loglevel := flag.String("loglevel", "0", "loglevel: klog level")
 	flag.Parse()
 
 	// Set the application's log level.
 	setLogLevel(*loglevel)
 
-	// Verify CLI flags.
-	if *srcPodName == "" {
```

Comment:
why did we move these checks to a new place?

Reply:
Retrieving the kubeconfig and coreclient is now done before these checks since it's needed fo discover the tableIDs, laeter on when we refactor with subcommands it will look better.